pkg/commands/artifact/run.go

```diff
@@ -609,6 +610,12 @@ func Exit(opts flag.Options, failedResults bool) {
 	}
 }
 
+func ExitOnEosl(opts flag.Options, m types.Metadata) {
```

Comment:
There is no need to export the function as it is not used somewhere else.

```suggestion
func exitOnEosl(opts flag.Options, m types.Metadata) {
```